### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by rem7i** [WebDevStudios/nextjs-wordpress-starter#956] [General]
All six WDS Headless plugins are NOT working on PHP 8+
You can install them, but CANNOT activate at all.
Are you working to fix this or  we MUST downgrade the PHP Version?

**Comment by gregrickaby** [comment]
@rem7i Thanks for the heads up. I've created an issue: https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/958
